### ISSUE TITLE
FocusTrapCallout Variant from documentation site error, "Ensures aria-hidden elements do not contain focusable elements"

### ISSUE THREAD

**Issue by wsaballos** [microsoft/fluentui#15309]
### Environment Information

- **Package version(s)**: 7.140.0
- **Browser and OS versions**: Chrome (85.0.4183.121) & MacOS 10.15.6 

### Describe the issue:

Accessibility Insights is reporting a failure with the FocusTrapCallout Variant in the documentation site. I can't reproduce the error on the documentation site, but when exported to Codepen I start to receive "Ensures aria-hidden elements do not contain focusable elements" once the focustrap is opened.

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/wsaballos/pen/JjXqEdm?editable=true%3Dhttps%3A%2F%2Fdeveloper.microsoft.com%2F

#### Actual behavior:

When using `enableAriaHiddenSiblings: true` Accessibility Insights is reporting an error when focus trap is open.

I have tried a barebones example with the Modal while using `enableAriaHiddenSiblings: true` and Accessibility Insights does not report an error with that example.

#### Expected behavior:

I believe that `ms-Callout-container` should receive aria-hidden="true" once the focustrap is opened. 

### Documentation describing expected behavior

<img width="725" alt="Screen Shot 2020-09-29 at 12 04 23 PM" src="https://user-images.githubusercontent.com/8570984/94604040-f455d280-024b-11eb-96f3-87b6d5657109.png">
